packages/siopv2-oid4vp-op-auth/src/machine/Siopv2Machine.ts

```diff
@@ -38,6 +39,19 @@ const Siopv2HasAuthorizationRequestGuard = (_ctx: Siopv2MachineContext, _event:
   return authorizationRequestData !== undefined
 }
 
+const Siopv2CanDoGetSelectableCredentialsGuard = (_ctx: Siopv2MachineContext, _event: Siopv2MachineEventTypes): boolean => {
```

Comment:
This is a pretty bad name. CanDoGetSelecatable.....

Reply:
Okay, initially I had Siopv2HasSelectableCredentialsAndContactGuard which I reverted to now. But it's a bit long and was looking for something smaller. When I add another two to more conditions to Siopv2HasSelectableCredentialsAndContactGuard  it's going to be 
Siopv2HasSelectableCredentialsAndContactAndOtherThingAndYetAnotherThingGuard  ?

Reply:
Well it should not be talking about the other precondintions to begin with. The guard is responsible for whether VP is used or SIOP only